addons/account/views/account_move_views.xml

```diff
@@ -779,10 +779,10 @@
                                         <field name="sequence" invisible="1"/>
                                         <field name="tax_line_id" invisible="1"/>
                                         <field name="tax_group_id" invisible="1"/>
-                                        <field name="tax_repartition_line_id" invisible="1"/>
+                                        <field name="tax_repartition_line_id" invisible="1" force_save="1"/>
                                         <field name="tag_ids" invisible="1"/>
-                                        <field name="tax_base_amount" invisible="1"/>
-                                        <field name="tax_exigible" invisible="1"/>
+                                        <field name="tax_base_amount" invisible="1" force_save="1"/>
+                                        <field name="tax_exigible" invisible="1" force_save="1"/>
```

Comment:
since we made these 3 fields readonly, it is not correctly updating `line_ids`. Try this case after removing `force_save` for these fields,
1) create an invoice with one line having, say, 21% tax, and save the invoice
2) modify above invoice line and set 10% tax and save the invoice

result: `line_ids` should have been calculated considering 10% tax only, instead it keeps 21% tax and also considers 10% tax as well.